README.md

```diff
@@ -136,7 +136,7 @@ Congrats! Now you have ThreadDB running locally.
 
 Note the various configuration values shown in the output above. These can be modified with environment variables show below.
 
--   ***`THRDS_REPO`***: Repo location. `.threads` by default.
+-   ***`THRDS_REPO`***: Repo location. Mandatory when launching from docker compose. `.threads` by default otherwise. 
```

Comment:
This seems to complicating things. Maybe not mentioning the default value is better?

Reply:
Hmm, yeah that is confusing. I like your idea of removing the mention of default.

Reply:
Done